kiwicom/orbit
# chore: add tailwind folder as temporary storage for refactored compon…

As we agreed, creating the temporary folder for storing refactored TW components

To test:
- run *yarn components build* and check the output of the es/lib folders 
- or look at the result of compressed-size-action


 Storybook: https://orbit-mainframev-chore-tailwind-folder.surge.sh
